e2e/dpos-downtime.toml

```diff
@@ -121,3 +126,11 @@
   RunCmd = "{{ $.LoomPath }} dpos3 check-delegation {{index $.NodeAddressList 2}} {{index $.AccountAddressList 0}}"
   Condition = "contains"
   Expected = ["2000000000000000000"]
+
+# Node 1 staked 1.25M in order to become a validator, if the node was properly
+# slashed, the node's self-delegation should now be a bit less than 1.25M so the
+# figure "125" should not be present in the response.
+[[TestCases]]
+  RunCmd = "{{ $.LoomPath }} dpos3 check-delegation {{index $.NodeAddressList 1}} {{index $.NodeAddressList 1}}"
+#  Condition = "excludes"
+#  Excluded = ["125"]
```

Comment:
Why is this commented out?

Reply:
Ah ok, flaky tests presumably. @pathornteng can probably fix them up.

Reply:
yep, flakey tests -- supplemented w/ unit tests.